tests/test_experiments.py

```diff
@@ -75,6 +75,9 @@ def coco_cfg(tmp_path) -> Dict:
 
 
 @RunIf(sh=True)
+@pytest.mark.skipif(
+    not _IN_CI and not _HAS_TORCHVISION, reason="test requires that torchvision is installed"
+)
```

Comment:
I wonder if it's just better to not check `_IN_CI` here? Seems to get redundant very fast. What we should check is that when `CI=1`, then make sure that `_HAS_TORCHVISION=1`, `_HAS_TIMM=1`, etc.

Reply:
Good idea. I added a standalone test for dependency on CI and purged the CI checks elsewhere.